compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -1305,6 +1310,8 @@ impl<'a> Parser<'a> {
             self.parse_lit_expr(attrs)
         } else if self.check(&token::OpenDelim(Delimiter::Parenthesis)) {
             self.parse_tuple_parens_expr(attrs)
+        } else if self.check(&token::OpenDelim(Delimiter::Invisible { skip: false })) {
```

Comment:
```suggestion
        } else if self.check(&token::OpenDelim(Delimiter::Invisible { .. })) {
```
If we see it in the parser then it wasn't skipped by definition.
It can be kept as a kind of assert, but it will become inconvenient when more flags are added to `Delimiter::Invisible`.

Reply:
This is an expression, not a pattern, so the field must be provided.